src/django_idom/templatetags/idom.py

```diff
@@ -16,13 +16,15 @@
 register = template.Library()
 
 
-@register.inclusion_tag("idom/view.html")
+@register.inclusion_tag("idom/component.html")
```

Comment:
Looks like you changed this without changing the name of file itself.

Reply:
Just checked, I did push the [file name change](https://github.com/idom-team/django-idom/blob/bbf3948d696f505ee459ee29f17c16151bc35197/src/django_idom/templates/idom/component.html).

Also viewable in the `files changed` tab on this github PR.

Let me know if I'm misunderstanding.